documentation/bootstrap_upgrade_notes.md

```diff
@@ -0,0 +1,47 @@
+# Bootstrap upgrade notes
```

Comment:
This should have a YAML header instead of an H1, such as:

``` yaml
---
layout: default
title: "Puppet Server: Bootstrap Upgrade Notes"
canonical: "/puppetserver/latest/bootstrap_upgrade_notes.html"
---
```